nixos/modules/services/web-apps/pretix.nix

```
@@ -479,7 +479,9 @@ in
           CapabilityBoundingSet = [ "" ];
           DevicePolicy = "closed";
           LockPersonality = true;
-          MemoryDenyWriteExecute = true;
+          # Secuity note: This option is incompatible with e.g. JIT execution engines such as
+          # the lovely java vm that pdftk depends on
+          MemoryDenyWriteExecute = false;
```

```suggestion
          MemoryDenyWriteExecute = false; # required by pdftk
```

I don't think we need to explain the option again, and we also shouldn't merge my snarky comment into this.